App.js

```diff
@@ -1,24 +1,29 @@
 import React from "react";
 import { StyleSheet, Platform, Image, Text, View } from "react-native";
 import {
-	createSwitchNavigator,
-	createStackNavigator,
-	createAppContainer
+    createSwitchNavigator,
+    createStackNavigator,
+    createAppContainer,
 } from "@react-navigation/stack";
 
 import * as firebase from "firebase";
+import "firebase/firestore";
+
 var firebaseConfig = {
-	apiKey: "AIzaSyBLJ-dDm-yr9YouJny1tf7F5Z4CTgrtgsA",
-	authDomain: "vroomwash-f6735.firebaseapp.com",
-	databaseURL: "https://vroomwash-f6735.firebaseio.com",
-	projectId: "vroomwash-f6735",
-	storageBucket: "vroomwash-f6735.appspot.com",
-	messagingSenderId: "465276854623",
-	appId: "1:465276854623:web:8f9568eda142f82da996ea"
+    apiKey: "AIzaSyBLJ-dDm-yr9YouJny1tf7F5Z4CTgrtgsA",
+    authDomain: "vroomwash-f6735.firebaseapp.com",
+    databaseURL: "https://vroomwash-f6735.firebaseio.com",
+    projectId: "vroomwash-f6735",
+    storageBucket: "vroomwash-f6735.appspot.com",
+    messagingSenderId: "465276854623",
+    appId: "1:465276854623:web:8f9568eda142f82da996ea",
 };
 // Initialize Firebase
 firebase.initializeApp(firebaseConfig);
 
+// define the firestore database
+const dbh = firebase.firestore();
```

Comment:
You are not using this... great idea to just start dbh and do all the functions next. Use it or axe it. :)

Reply:
Axed